core/settings.py

```diff
@@ -48,6 +48,26 @@
 if RENDER_EXTERNAL_HOSTNAME:    
     ALLOWED_HOSTS.append(RENDER_EXTERNAL_HOSTNAME)
 
+#Secure Cookies
+#Ensure cookies are only sent over HTTPS
+SESSION_COOKIE_SECURE = True
+
+# Prevents JavaScript from accessing session cookies
+SESSION_COOKIE_HTTPONLY = True
+
+#Mitigate CSRF attacks by restricting cross-origin cookie sharing
+SESSION_COOKIE_SAMESITE = 'Strict'
+
+#Ensure CSRF cookies are sent over HTTPS only
+CSRF_COOKIE_SECURE = True
+
+#Enhance CSRF protection
+CSRF_COOKIE_SAMESITE = 'Strict'
+
+#Ensure DEBUG is set to False in production to avoid sensitive information exposure
+DEBUG = False
+
```

Comment:
Do not commit changing DEBUG to false
please update the pull request setting DEBUG back to true

Reply:
Thanks for the feedback. Requested change has been updated.